### ISSUE TITLE
Incomplete url patterns in Priority_Todo_List project folder

### ISSUE THREAD

**Issue by Varun487** [acmpesuecc/Todo_List_In_Django#3]
The urls.py in the Priority_Todo_List project is incomplete. You must connect the url patterns in Todo app with the url patterns of the project folder

**Comment by sawyer1997** [comment]
@Varun487 I think I can do, let me know if I can go ahead.

**Comment by Varun487** [comment]
Hey @sawyer1997, are you part of HackNight 2020 (organised by PES)? If yes, just enter the discord server and ping me in the Todolist channel

**Comment by sawyer1997** [comment]
No, I am not part of HackNight. Is it okay if I can still go ahead? @Varun487

**Comment by Varun487** [comment]
Sure, go ahead! Just asking. @sawyer1997

**Comment by sawyer1997** [comment]
Please assign this to me. @Varun487
